google/cloud/vision/color.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Image properties class representation derived from Vision API response."""
+
+
+class ImagePropertiesAnnotation(object):
+    """Representation of image properties
+
+    :type colors: list
+    :param colors: List of
+                   :class:`~google.cloud.vision.color.ColorInformation`.
+    """
+    def __init__(self, colors):
+        self._colors = colors
+
+    @classmethod
+    def from_api_repr(cls, response):
+        """Factory: construct ``ImagePropertiesAnnotation`` from a response.
+
+        :type response: dict
+        :param response: Dictionary response from Vision API with image
+                         properties data.
+
+        :rtype: :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`.
+        :returns: Populated instance of ``ImagePropertiesAnnotation``.
+        """
+        colors = [ColorInformation.from_api_repr(color) for color in
+                  response['dominantColors']['colors']]
+        return cls(colors)
+
+    @property
+    def colors(self):
+        """Colors in an image.
+
+        :rtype: list of :class:`~google.cloud.vision.color.ColorInformation`
+        :returns: Populated list of ``ColorInformation``.
+        """
+        return self._colors
+
+
+class Color(object):
+    """Representation of RGBA color information.
+
+    :type red: int
+    :param red: The amount of red in the color as a value in the interval
+                [0, 1].
```

Comment:
Ha! I copied it from the API docs [here](https://cloud.google.com/vision/reference/rest/v1/images/annotate#color).

Can't believe I missed that.